web/profiles/joinup/joinup.info.yml

```diff
@@ -57,17 +58,21 @@ config_devel:
   - block.block.seven_page_title
   - block.block.seven_primary_local_tasks
   - block.block.seven_secondary_local_tasks
+  - block.block.userlogin
   - core.date_format.fallback
-  - core.date_format.html_datetime
   - core.date_format.html_date
+  - core.date_format.html_datetime
   - core.date_format.html_month
   - core.date_format.html_time
   - core.date_format.html_week
-  - core.date_format.html_yearless_date
   - core.date_format.html_year
+  - core.date_format.html_yearless_date
   - core.date_format.long
   - core.date_format.medium
   - core.date_format.short
+  - core.entity_form_display.rdf_entity.collection.propose
```

Comment:
This is here because the "Propose" mechanism is not exclusive to collections, in the future we will also be able to propose interoperability solutions etc.

To facilitate this, I've created a new form display mode on the RDF entity that is called "propose" (ref. `joinup_entity_type_alter()`). In the future we will add the form displays for interoperability solutions alongside this.